src/Schema/Events/ImpressionTracking.ts

```diff
@@ -80,3 +80,30 @@ export interface TooltipViewed {
   context_owner_type: PageOwnerType
   type: string
 }
+
+/**
+ * A user sees a an error message
+ *
+ * This schema describes events sent to Segment from [[ErrorMessageViewed]].
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "errorMessageViewed",
+ *    context_owner_type: "orders",
+ *    context_owner_id: "order-id",
+ *    title: "This is an error",
+ *    message: "Error message",
+ *    error_code: "charge_authorization_failed"
```

Comment:
Can we add a "route" field to track which route the error occurs on, such as "payment", "shipping", "offer", "review", etc?

Reply:
Oh oops, I didn't see the conversation! that sounds good thanks!